build/configs/sidk_s5jt200/README.md

```diff
@@ -8,6 +8,7 @@ Samsung IoT Development Kit for S5JT200 chipset.
 > [Environment Set-up](#environment-set-up)  
 > [How to program a binary](#how-to-program-a-binary)  
 > [ROMFS](#romfs)  
+> [Using GDB](#gdb)
```

Comment:
I've checked it at your repo. It is not working. Let's modify it as I mentioned.